crates/bevy_pbr/src/light.rs

```diff
@@ -92,6 +92,9 @@ impl Default for PointLightShadowMap {
 /// approximation for light sources VERY far away, like the sun or
 /// the moon.
 ///
+/// By default, the light shines along the negative-Z axis and the local
+/// rotation adjusts its direction.
```

Comment:
Sorry for the bikeshedding, I had this as a pending suggestion apparently so I thought I'd throw it out there to see which is preferred.
```suggestion
/// The light shines along the forward direction of the entity's transform. With a default transform
/// this would be along the negative-Z axis.
```

Reply:
Sure, let's use that. 

Feedback from the group, there are a lot of places I'd like to add documentation and I think it's better to have progress over perfection to get some more coverage. Here I think most most developers like myself just need the default direction and can reason the transform affects it. I feel like I'm missing this significance of this "forward direction" and don't know if it's a common term used by bevy folks. I've never used an API or rendering system with two-way directional lights so it seems like an odd clarification.

Reply:
`Transform` has a `forward()` method on it. That is the forward direction.